apps/site/src/components/pages/auth-wall.tsx

```diff
@@ -1,15 +1,14 @@
-import { NextPage } from "next";
 import { useRouter } from "next/router";
-import { FunctionComponent } from "react";
 
```

Comment:
@benwerner01 Minor tweak while we're working on this page: please can we increase the font-weight of the `PLANNED` labels next to certain features, everywhere they appear on the site (including here on the upgrade checkout pages?)

They're not meant to be as thin as they're appearing. See Figma. Cheers!

Reply:
Adjusted these in https://github.com/blockprotocol/blockprotocol/pull/1152/commits/0f190302c26add3eda765e288693aabaaa314418

<img width="320" alt="image" src="https://user-images.githubusercontent.com/42802102/224303296-6bd83659-082a-4d7c-9f3b-6607f472f021.png">